src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -195,7 +195,7 @@ public Orientation Orientation
         /// <inheritdoc/>
         protected override Size ArrangeOverride(Size finalSize)
         {
-            UpdateIndicator(finalSize);
+            UpdateIndicator();
```

Comment:
I think at this point we have previous bounds, before arranging. 
If you set ProgressBar page as a starting page in control catalog, indicator won't be rendered.

@timunie do you want to check it? Otherwise, I can do it later this week.

Reply:
I've introduced the bug, so I should try to fix it. If I find a solution, I'll open a new PR for this.